universal-application-tool-0.0.1/app/views/LoginForm.java

```diff
@@ -61,6 +61,8 @@ private ContainerTag mainContent(Messages messages) {
         this.layout
             .viewUtils
             .makeLocalImageTag("ChiefSeattle_Blue")
+            .withAlt("City of Seattle Logo")
```

Comment:
(blocking) these should be i18nized right?

Reply:
Not worth it imo - they are not going to be read aloud, due to `aria-hidden: true`.